discord/ext/commands/core.py

```diff
@@ -94,10 +93,7 @@
     'is_owner',
     'is_nsfw',
     'has_guild_permissions',
-    'bot_has_guild_permissions',
```

Comment:
I'm assuming this was intended? So that people could use `commands.slash_command` etc.

Reply:
I think this was accidentally removed when I was reverting some changes

Reply:
Although, I think user might be better doing
`from discord.app import slash_command` it works too.